Pipfile

```diff
@@ -8,6 +8,7 @@ verify_ssl = true
 [packages]
 discord = "*"
 python-dotenv = "*"
+PyNaCl = "*"
```

Comment:
what's this new dependency for? doesn't seem to be used anywhere?

Reply:
Discord complains about it not existing for voice connecting reasons so I've added it for when we add voice connectability in the future and to remove the warning message